Wrappers/Python/cil/optimisation/utilities/StepSizeMethods.py

```diff
@@ -177,6 +178,9 @@ def __init__(self, initial, mode='short', stabilisation_param="auto"):
         '''
  
         self.mode=mode
+        self.current_mode = mode
+        if self.current_mode == "alternate":
```

Comment:
If you change this to a boolean e.g. `is_short` then in `get_step_size` it simplifies for if statement and lets you toggle the state in one line at the end of the call.

Reply:
Done